src/ui/MessageInput/index.tsx

```diff
@@ -230,6 +277,7 @@ const MessageInput = React.forwardRef<HTMLInputElement, MessageInputProps>((prop
       }
       setIsInput(textField?.textContent?.trim().length > 0);
       setHeight();
+      moveCursorToEnd(textField);
```

Comment:
what does this do? does this create some weird behavior with inserting emojis in the middle of text? i fixed that here https://github.com/gathertown/gather-town/pull/25302/files#diff-4c9b26b8a63139d083fba83939892f7a97c05b9de7433e5bbe5489ac32fc0c5dR78-R92

Reply:
Ahh, I'll add a `// fork note` comment here, but this particular call is to force the cursor to the end when editing or after loading a draft message.

Reply:
ohhh nice i had to add this in the gather-browser text input component

https://github.com/gathertown/gather-town/pull/25302/files#diff-4c9b26b8a63139d083fba83939892f7a97c05b9de7433e5bbe5489ac32fc0c5dR67-R77